MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```
@@ -28,9 +30,11 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 	func display(_ viewModel: FeedLoadingViewModel) {
 		if viewModel.isLoading {
 			refreshControl?.beginRefreshing()
+			errorView?.hideMessage()
```

The Presenter should decide when to hide/show errors. But here the view controller is deciding to hide the message when the ViewModel.isLoading is `true`.

But the view controller should change the error view state based on the error message model (not the loading state, because the presenter could decide to show another error while still loading!). 

So the presenter will send an error view model to the view controller. The view controller should render itself based on that view model state.

So if there's an error message, it shows the error. Else, it hides the error.

This way, the controller doesn't need to decide when to hide/show errors. It's easier to manage presentation state transitions when all changes are centralized in the presenter.